README.md

```diff
@@ -57,10 +57,90 @@ is installed:
 
 3. Hit F5 to build. The output will be in the "bin/Release" or "bin/Debug" directory.
 
-### Linux
-Open a terminal instance and put the following commands:
+## Linux
+1. Open a terminal instance and put the following commands:
+
+### Ubuntu
+![](https://assets.ubuntu.com/v1/ad9a02ac-ubuntu-orange.gif)
+```bash
+sudo apt update
+sudo apt install dirmngr gnupg apt-transport-https ca-certificates software-properties-common
+sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 3FA7E0328081BFF6A14DA29AA6A19B38D3D831EF
+```
+
+> for Ubuntu 20.10 - groovy
+```
+sudo apt-add-repository 'deb https://download.mono-project.com/repo/ubuntu stable-groovy main'
+```
+
+> for Ubuntu 20.04 - focal
+```
+sudo apt-add-repository 'deb https://download.mono-project.com/repo/ubuntu stable-focal main'
+```
+
+> for Ubuntu 19.10 - eoam
+```
+sudo apt-add-repository 'deb https://download.mono-project.com/repo/ubuntu stable-eoam main'
+```
+
+> for Ubuntu 19.04 - disco
+```
+sudo apt-add-repository 'deb https://download.mono-project.com/repo/ubuntu stable-disco main'
+```
+
+> for Ubuntu 18.10 - cosmic
+```
+sudo apt-add-repository 'deb https://download.mono-project.com/repo/ubuntu stable-cosmic main'
+```
+
+> for Ubuntu 18.04 - bionic
+```
+sudo apt-add-repository 'deb https://download.mono-project.com/repo/ubuntu stable-bionic main'
```

Comment:
You can use a single command here:
```
sudo apt-add-repository "deb https://download.mono-project.com/repo/ubuntu stable-`lsb_release -cs` main"
```

Just to make sure `lsb_release` is installed, add this package `lsb-release` to the command where you install other required packages.

Reply:
completely forgot about that, thanks